internal/command/jobs/printers.go

```diff
@@ -110,6 +113,59 @@ func printJobDetails(cfg *config.Job, out io.Writer, job *models.Job) error {
 	return nil
 }
 
+func waitForJob(cfg *config.JobSubmit, out io.Writer, jobName string) error {
+
+	fmt.Fprintf(out, "Waiting for job execution\n")
+
+	looped := false
+
+	for {
+		j, err := getJob(cfg.Job, jobName)
+		if err != nil {
+			return err
+		}
+
+		switch j.Status.Phase {
+		case "completed":
+			os.Exit(int(j.Status.Attempts[0].State.Completed.ExitCode))
```

Comment:
Not sure how we end up with `ExitCode int64` in the SDK, but both in the operator and the SaaS, the exit code of a given job may be nil:

https://github.com/signadot/signadot/blob/7740cc65653ed190cf089bd463df11e7aa9e7563/sandboxes/api/control/signadotv2/jobs/status.go#L56-L60

I would double check, I think there is the risk of treating `nil` exit codes as `0` (when a `nil` exit code, means there was an error running the job).